### ISSUE TITLE
Circle's radius

### ISSUE THREAD

**Issue by satuelisa** [KevinMichelle/Vision_FIME_2015#2]
I find the way in which you fix the radius rather risky and it seems to be the reason why your votes are all weird and mostly off the intended center. If you do want to use a fixed value, I recommend computing the bounding box, checking for approximate symmetry (if it is not a square, your object is either not a circle or not a complete one), and then using the half of the side length. I would, however, vote within a discretized range taking values at least from the half of the minimum side length of the bounding box to the half of the maximum side length of the bounding box.

**Comment by KevinMichelle** [comment]
Yeah, I think that my implementation is not so good as I would like. I will find better ways of do it to change it.
